rdbms/models/telephones.py

```diff
@@ -0,0 +1,11 @@
+from manage import Base
+from sqlalchemy import Column, Integer, ForeignKey
+
+
+class Telephones(Base):
+    __tablename__ = 'telephones'
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    telephone_number = Column(Integer, nullable=False)
```

Comment:
Phone numbers may contain special characters such as - and (). Use an appropriate data type two account for the variability of phone number formats.